lib/trento_web/controllers/fallback_controller.ex

```diff
@@ -0,0 +1,35 @@
+defmodule TrentoWeb.FallbackController do
+  use TrentoWeb, :controller
+
+  alias TrentoWeb.ErrorView
+
+  def call(conn, {:error, {:bad_request, errors = %{}}}) do
+    conn
+    |> put_status(:bad_request)
+    |> render_errors(errors)
+  end
+
+  def call(conn, {:error, {:bad_request, reason}}) do
+    conn
+    |> put_status(:bad_request)
+    |> render_error(reason)
+  end
+
+  def call(conn, {:error, {:internal_error, reason}}) do
+    conn
+    |> put_status(:internal_server_error)
+    |> render_error(reason)
+  end
+
```

Comment:
I don't know if we should have a last pattern match with something more generic:
```
  def call(conn, {:error, reason}) do
```
But I guess that we are on control of sending these values, so we should be fine.
Just we need to be careful as the fallback action is not that obvious and we could en up sending something that is not matched here

Reply:
Yes, it could happen, I tried to match all the errors we produce directly, the rest of the errors should be catched by other exception handlers, like ecto or somehting generic about 500